internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -589,9 +611,7 @@ func resourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Di
 			return diag.FromErr(fmt.Errorf(ErrorClusterAdvancedSetting, "replication_specs", clusterName, err))
 		}
 
-		clusterResp = convertClusterDescToLatestExcludeRepSpecs(clusterOldSDK)
-		clusterResp.ConfigServerManagementMode = cluster.ConfigServerManagementMode
-		clusterResp.ConfigServerType = cluster.ConfigServerType
+		clusterResp = cluster
```

Comment:
clusterResp (associated to latest `connV2` model) is already available, no need to convert old model.